lib/dialects/sqlite/data-types.js

```diff
@@ -97,11 +97,41 @@ FLOAT.prototype.toSql = function() {
   return NUMBER.prototype.toSql.call(this);
 };
 
+var DOUBLE = function(length, decimals) {
```

Comment:
The difference is the order of the parameters.

E.g.: 
 'REAL(11,12) UNSIGNED',
sqlite: 'REAL UNSIGNED(11,12)',